network.go

```diff
@@ -195,6 +195,12 @@ func (n *network) CreateEndpoint(name string, options ...EndpointOption) (Endpoi
 	if name == "" {
 		return nil, ErrInvalidName(name)
 	}
+
+	e, _ := n.EndpointByName(name)
```

Comment:
I think we discussed enough about whether to use galvanized aluminium or iron for that bike shed ;-)

If you feel discarding the error or the published behaviour of the API is not a problem, because you know its current implementation, what can I say ?
If we later change the walk function to return an error (we talked about that), are we going to revisit all the code where the EndpointByName() is being called ? In other packages too ?